tests/virt_autotest/update_package.pm

```diff
@@ -54,9 +55,12 @@ sub run {
         set_serial_console_on_vh('', '', 'kvm') if (check_var("HOST_HYPERVISOR", "kvm") || check_var("SYSTEM_ROLE", "kvm"));
     }
     update_guest_configurations_with_daily_build();
+    if (is_remote_backend && check_var('ARCH', 'aarch64') && (get_var("VIRT_PRJ2_HOST_UPGRADE") || get_var("VIRT_PRJ4_GUEST_UPGRADE"))) {
+        my $arm64console = get_var("SERIALDEV", "ttyAMA0");
+        assert_script_run("sed -irn \"s/console=ttyAMA0/console=$arm64console/g\" /usr/share/qa/virtautolib/lib/vh-update-lib.sh");
+    }
 }
```

Comment:
would you please make this change to the library call, lib/ipmi_backend_utils.pm::setup_console_in_grub, so that all calling to this api can save this effort?

Reply:
Done